lib/artsy-eventservice/artsy/event_service/rabbitmq_connection.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'bunny'
+
+module Artsy
+  module EventService
+    module RabbitMQConnection
+      # get a new RabbitMQ Client
+      def create_conn
+        Bunny.new(rabbitmq_url, **bunny_params)
+      end
+
+      # Connect, do something and close the connection
+      def connect_to_rabbit
```

Comment:
Yea so ideally, we want to open one connection per process, and then 1 channel per thread.  Further reading: 
- http://rubybunny.info/articles/connecting.html
- http://stackoverflow.com/questions/10407760/is-there-a-performance-difference-between-pooling-connections-or-channels-in-rab

I wonder how we could enforce this though, given that this is a shared library, imported into different applications (i.e. some Rails projects using Unicorn, where the connection should be set up in the `:after_fork` block, and some projects using Puma, which is a threaded process worker model) - thoughts @ashkan18 ?